src/com/oltpbenchmark/api/Worker.java

```diff
@@ -518,7 +526,7 @@ protected final TransactionExecutionState doWork(SubmittedProcedure pieceOfWork)
                                                ex.getClass().getSimpleName(), next, this.toString(),
                                                ex.getMessage(), ex.getErrorCode(), ex.getSQLState()), ex);
 
-		    if (Worker.wrkld.getDBType().shouldUseTransactions()) {
+		    if (Worker.wrkld.getDBType().shouldUseTransactions() && !conn.getAutoCommit()) {
```

Comment:
Worker.wrkld.getDBType().shouldUseTransactions() is always true for us..
So maybe remove that?